### ISSUE TITLE
Distillation loss

### ISSUE THREAD

**Issue by wxbxjtu** [facebookresearch/deit#116]
Dear all,
         I have a question. Why in 
         distillation_loss = F.kl_div(
                F.log_softmax(outputs_kd / T, dim=1),
                F.log_softmax(teacher_outputs / T, dim=1),
                reduction='sum',
                log_target=True
            ) * (T * T) / outputs_kd.numel()
         distillation_loss is divided by outputs_kd.numel() and not the batchsize here? Thank you!

**Comment by TouvronHugo** [comment]
Hi @wxbxjtu ,
Thanks for your question,
See [issues 61](https://github.com/facebookresearch/deit/issues/61)  for the answer.
Best,
Hugo

**Comment by wxbxjtu** [comment]
> Hi @wxbxjtu ,
> Thanks for your question,
> See [issues 61](https://github.com/facebookresearch/deit/issues/61) for the answer.
> Best,
> Hugo

Thank you for your answer.
